src/Button/index.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FC, PropsWithChildren } from "react";
+
+import "./index.scss";
+import clsx from "clsx";
+
+export interface ButtonProps {
+  /**
+   * The button's size.
+   */
+  size?: "sm" | "lg";
+
+  /**
+   * The button's color scheme.
+   */
+  color?: "black" | "white" | "dark-blue" | "red" | "blue-a";
```

Comment:
I think it's more important to expose the _kind_ of button (e.g., primary, secondary, danger) than the color: this makes it a lot easier for folks to know which one to use. As an alternative, we can follow the [USWDS example](https://designsystem.digital.gov/components/button/) where the styles are constrained instead by primary/secondary/cool/warm.

The important point is that we should be constraining button styling to make it easier to answer the question of "which button do I use?".